components/kyma-operator/Dockerfile

```diff
@@ -27,7 +27,7 @@ WORKDIR /app
 # Install certificates
 #
 
-RUN apk update && apk add ca-certificates git && rm -rf /var/cache/apk/*
+RUN apk update && apk add ca-certificates && rm -rf /var/cache/apk/*
```

Comment:
We can not remove it, it was added when we added feature to install components from remote locations(including git repos). `go-getter` needs this binary